node_modules/oae-admin/tenants/tenants.html

```diff
@@ -121,8 +42,99 @@
             </div>
         </div>
     {/if}
+    {var isOnGlobalAdminServer = context.isGlobalAdminServer || context.isTenantOnGlobalAdminServer}
+    {if context.isGlobalAdminServer}
+        <div id="tenants-search">
```

Comment:
Rather than defining a new search form, we should use the search form that's available through the `listHeader` template